kubernetes/omsagent.yaml

```diff
@@ -379,13 +379,19 @@ spec:
               memory: 225Mi
           env:
             # azure devops pipeline uses AKS_RESOURCE_ID and AKS_REGION hence ensure to uncomment these
+            - name: AKS_CLUSTER_NAME
+              value: "VALUE_AKS_CLUSTER_NAME"
             - name: AKS_RESOURCE_ID
               value: "VALUE_AKS_RESOURCE_ID_VALUE"
             - name: AKS_REGION
               value: "VALUE_AKS_RESOURCE_REGION_VALUE"
+            - name: FBIT_SERVICE_FLUSH_INTERVAL
+              value: "15"
+            - name: AKS_NODE_RESOURCE_GROUP
```

Comment:
I dont think we use this anywhere in our code. why do we need that?

Reply:
Removed this